.github/workflows/integration-tests.yml

```diff
@@ -76,7 +76,7 @@ jobs:
           timeout_minutes: 15
           max_attempts: 3
           retry_on: error
-          command: npm run test:ci:vitest:integration -- --shard=${{ matrix.shard }}
+          command: npm run test:integration -- --coverage --shard=${{ matrix.shard }}
```

Comment:
huh, why do we enable coverage only for integration and not unit? 🤔 I guess were aren't reporting it anywhere anyway so...

Reply:
We did enable coverage for both:

https://github.com/netlify/cli/pull/7152/files#diff-4761c5069b46ce96f49d74476da61f43ced3abc73d22254cd95a2be0e250a593R79

and still do enable coverage for unit tests:

https://github.com/netlify/cli/pull/7152/files#diff-cd0effce06f425599e96952f4d5f684641a683f46d9909e7c74a9f8b2f03ccd3R48

But yeah, we don't do anything meaningful with the reports.